pkg/types/libvirt/platform.go

```diff
@@ -33,4 +33,11 @@ type Network struct {
 	// +kubebuilder:default="tt0"
 	// +optional
 	IfName string `json:"if,omitempty"`
+
+	// DnsmasqOptions is the dnsmasq options to be used when installing on
+	// libvirt. This is useful to force domains to a specific IP or to a
+	// loadbalancer.
+	//
+	// +optional
+	DnsmasqOptions map[string]string `json:"dnsmasqOptions,omitempty"`
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#lists-of-named-subobjects-preferred-over-maps

Reply:
so reading the comments in the link, it looks like we prefer named subobjects over maps for better ux and readability. would something like this suffice? 

```
platform:
     libvirt:
       network:
         dnsmasqOptions:
         - optionName: "address"
           optionValue: "/.apps.tt.testing/192.168.126.51"
         if: tt0
```

Reply:
Yes, although the "option" part of `optionName` and `optionValue` are a bit redundant with `dnsmasqOptions`. How would you feel about simply `name` and `value`?

Reply:
makes sense, respun with the changes.